### PR TITLE
[UTOPIA-1403] MPO Review E2E

### PR DESCRIPTION
**Pull request by JieunSon96**
# Description

This PR includes the following proposed change(s):

- [UTOPIA-1403](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-1403) Added missing scenery 
- Changed Chromedriver to Firefox because of the Keycloak logout issue
- Added README.md for running a single feature file

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [x] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readable 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/.eslintignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ coverage/*
 /src/types/
 
 .eslintrc.js
+wdio.conf.js
```
Comment (BradyMitch) [inline src/frontend/.eslintignore:14]:
```suggestion
wdio.conf.js

```

File: src/frontend/test/features/pageobjects/fillPPQ.page.js [hunk 2]
```diff
@@ -1,83 +1,81 @@
...
-        {
-          var newlink=randomLink+i+"]/../label/input";
-          const buttonElement= await $(newlink);
-          await buttonElement.waitForClickable();
-          await buttonElement.click();
-          
-        }
   }
 
+  async clickCheckbox() {
```
Comment (BradyMitch) [inline src/frontend/test/features/pageobjects/fillPPQ.page.js:44]: I'm confused as to what this does

File: src/frontend/test/features/pageobjects/fillPPQ.page.js [hunk 3]
```diff
@@ -1,83 +1,81 @@
...
+   * e.g. to login using username and password
+   */
 
-  
-    
-    async selectCheckbox (checkName) {
-        const h1=link + checkName + "')]/../input";
-        const headerElement=await $(h1);
-        await headerElement.waitForDisplayed();
+  async selectCheckbox(checkName) {
```
Comment (BradyMitch) [inline src/frontend/test/features/pageobjects/fillPPQ.page.js:31]: Im confused how this works in the else block and whats going on between selectCheckbox and clickCheckbox

Comment (JieunSon96) [reply src/frontend/test/features/pageobjects/fillPPQ.page.js:31]: > Im confused how this works in the else block and whats going on between selectCheckbox and clickCheckbox

common function method like button, href, etc or selectors are managed under the page.js file so as not to duplicate the code and other sub pages extends the page.js to use the common methods. selectCheckbox function to handle the parameter of the checkbox name when we check the specific checkbox step under the feature file but the clickCheckbox function is for the random checkbox scenarios(generating a random number and selecting any random input checkbox).

File: src/frontend/test/features/pageobjects/fillPPQ.page.js (hunk 2 above)
Comment (JieunSon96) [reply src/frontend/test/features/pageobjects/fillPPQ.page.js:44]: > I'm confused as to what this does

I have answered together below comments

File: src/frontend/test/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+## Running a Single Feature File
+
+To run a single feature file, follow these steps:
+
+1. Identify the feature file you want to run, located in the ./test/features/featurefiles/e2e/ directory.
+
+2. Use the following command, replacing your-feature-file.feature with the name of the specific feature file you want to run:
+
+```bash
+npm run wdio -- --spec ./test/features/featurefiles/e2e/your-feature-file.feature
```
Comment (BradyMitch) [inline src/frontend/test/README.md:18]: Love this! Thanks for adding :)

File: src/frontend/test/features/pageobjects/submitPIA.page.js [hunk 5]
```diff
@@ -1,185 +1,275 @@
...
-
-    async savedAt () {
-        console.log('Check Saved At functionality ');
-        await browser.pause(5000);
-        const savedAtButton = await $( "//span[contains(text(),'Saved at')]");
-        await savedAtButton.waitForDisplayed({ timeout: 2000 });
-        }  
-        
-    async CheckPara (paragraph) {
+  async clickCheckboxByName(checkboxName) {
```
Comment (BradyMitch) [inline src/frontend/test/features/pageobjects/submitPIA.page.js:259]: Probably better to do this by Id but this will work for now

**Review by BradyMitch — APPROVED** [review]
Great work!

**Pull request merged**